### PR TITLE
Analytics-api play

### PR DESCRIPTION
**Pull request by dcadams**

### CODE REVIEW

File: stanford/playbooks/analytics-api.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    CLUSTER_NAME: analytics-api
+  when: redeploy | default(False) | bool
+
+- name: Deploy analytics-api service
+  hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_todo_deploy:&tag_Deployment_{{ COMMON_DEPLOYMENT }}:&tag_Number_{{ CLUSTER_NUMBER }}"
+  gather_facts: True
+  become: True
+  vars:
+    CLUSTER_NAME: analytics-api
+    migrate_db: "yes"
```
Comment (stvstnfrd) [inline stanford/playbooks/analytics-api.yml]: Nit: single quotes for non-interpolated strings.

File: stanford/playbooks/analytics-api.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    disable_edx_services: false
+    ENABLE_DATADOG: False
+    ENABLE_SPLUNKFORWARDER: False
+    ENABLE_NEWRELIC: False
```
Comment (stvstnfrd) [inline stanford/playbooks/analytics-api.yml]: These three `ENABLE_*` variables default to `False`, so we should omit them here.

File: stanford/playbooks/analytics-api.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  vars_files:
+    - "{{ COMMON_DEPLOYMENT_DIR }}/common.yaml"
+    - "{{ COMMON_DEPLOYMENT_DIR }}/insights.yml"
```
Comment (stvstnfrd) [inline stanford/playbooks/analytics-api.yml]: Which variables does `data-api` actually depend on from this?

Typically, the roles are fairly well segmented and only need their prefixed vars,
`INSIGHTS_*` for `insights`, `ANALYTICS_API_*` for `data-api`, etc..

Comment (dcadams) [reply stanford/playbooks/analytics-api.yml]: I'm going to try removing it and see what happens.

File: stanford/stage/analytics-api.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/analytics-api.yml
+  vars:
+    CLUSTER_NUMBER: 1
+    COMMON_DEPLOYMENT: stage
```
Comment (stvstnfrd) [inline stanford/stage/analytics-api.yml]: Looks like your file is missing a trailing newline.

File: stanford/stage/analytics-api.yml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ansible-playbook
```
Comment (stvstnfrd) [inline stanford/stage/analytics-api.yml]: Should this file be called `analytics-api1.yml`, per existing convention?

**Pull request merged**